pkgs/development/tools/rust/humility/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, libusb
+, libftdi
+, cargo-readme
+, pkg-config
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "humility";
+  version = "unstable-09-15-2022";
```

The date should be in `YYYY-MM-DD` format.

```suggestion
  version = "unstable-2022-09-15";
```